package.json

```diff
@@ -60,11 +56,8 @@
     "file-loader": "^0.9.0",
     "highlight.js": "~9.14.0",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^24.9.0",
-    "jest-prop-type-error": "^1.1.0",
     "node-sass": "^4.5.0",
     "prettier": "^1.19.1",
-    "raf": "^3.4.0",
     "raw-loader": "^0.5.1",
     "react-dnd-test-backend": "^9.4.0",
     "react-dnd-test-utils": "^9.4.0",
```

Comment:
Can it be removed?

Reply:
we use it only in jest AFAIK

Reply:
There are some more module that may move to the tfm-test.

- `axios-mock-adapter`
- `react-dnd-test-backend`
- `react-dnd-test-utils`
- `react-redux-test-utils`
- `react-test-renderer`

Can we move them from here and move them to the tfm-test?

Reply:
`react-test-render` can be removed totally - https://github.com/theforeman/foreman/pull/7312